src/Ocean/SuperModels.jl

```diff
@@ -0,0 +1,209 @@
+module SuperModels
+
+using ClimateMachine
+
+using ...DGMethods.NumericalFluxes
+
+using ..HydrostaticBoussinesq:
+    HydrostaticBoussinesqModel, NonLinearAdvectionTerm
+
+using ..OceanProblems: InitialValueProblem
+using ..Domains: array_type, communicator
+using ..Ocean: FreeSlip, Impenetrable, Insulating, OceanBC, Penetrable
+using ..Ocean.Fields: SpectralElementField
+
+using ...Mesh.Filters: CutoffFilter, ExponentialFilter
+using ...Mesh.Grids: polynomialorder
+
+using ClimateMachine:
+    LSRK144NiegemannDiehlBusch,
+    OceanBoxGCMConfigType,
+    OceanBoxGCMSpecificInfo,
+    DriverConfiguration
+
+import ClimateMachine: SolverConfiguration
+
+#####
+##### Build default EarthParameters
+#####
+
+using CLIMAParameters: AbstractEarthParameterSet
+
+struct EarthParameters <: AbstractEarthParameterSet end
```

Comment:
Please read [the CLIMAParameters.jl docs](https://clima.github.io/CLIMAParameters.jl/dev/#Implementation-notes). These structs must live at the driver level to ensure that we don't accidentally have multiple (and potentially inconsistent) implementations of parameters. We can define and instantiate the structs at the top level and pass them down into the driver wherever they may be needed.

Reply:
We need a centralized default. Where can I grab it from?

Reply:
If we move forward with `AbstractModels` the logical place for a default parameter set will indeed be in the submodule top-level `Model.jl`. I'm happy to move things around until then if a different solution is better.

Reply:
Why can't you pass it down? It's done like that everywhere else.

Reply:
Yes. But we did, I think, agree on adding some convenience constructors to `CLIMAParameters.jl`, particularly to help with code like [this](https://github.com/CliMA/ClimateMachine.jl/blob/aj/precip_interface/experiments/AtmosLES/dycoms.jl#L28-L56)?

Reply:
I removed the default, so I'm resolving this.